v2/docs/ProvidersApi.md

```diff
@@ -0,0 +1,96 @@
+# \ProvidersApi
```

Comment:
should the `\` be there?

Reply:
Yes it marks it as a heading (ProvidersApi section vs the ProvidersListAdmin member of the section)